CharacterCreation/02_Classes.txt

```diff
@@ -427,34 +428,100 @@ Skill Proficiencies:
 * Spotting
 * Survival
 
+== Hunter's Dog ==
+
+    When your character ascends to level 3, you receive an attack dog. Maybe 
+you found the dog along your last mission, or maybe a dog was issued to you when
+you returned to home base (at the GM's choice). Every time you level up after
+you have your dog, your dog levels up as well as defined in the 
+"Class-Specific Documentation/Hunter's_Dog.txt" document. For simplicity, Dogs 
```

Comment:
I think this sentence can be condensed a bit. 
For instance "Once you have a dog, it will level up with you, following the rules found in....document.